lib/mongo.js

```diff
@@ -0,0 +1,74 @@
+const { MongoClient, ObjectId } = require('mongodb')
+
+const MONGO_DB = 'flossbank_db'
+const MAINTAINERS_COLLECTION = 'users'
+
+class Mongo {
+  constructor ({ config, log }) {
+    this.log = log
+    this.config = config
+    this.db = null
+    this.mongoClient = null
+  }
+
+  async connect () {
+    const mongoUri = await this.config.getMongoUri()
+    this.mongoClient = new MongoClient(mongoUri, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true
+    })
+    await this.mongoClient.connect()
+
+    this.db = this.mongoClient.db(MONGO_DB)
+  }
+
+  async close () {
+    if (this.mongoClient) return this.mongoClient.close()
+  }
+
+  async getPendingMaintainerPayouts ({ maintainerId }) {
+    // Grab all maintainers who have a payout method (ILP pointer or otherwise)
```

Comment:
shouldn't we filter out maintainer with no payment pointer since this lambda can only use ILP ?

Reply:
Idk why I commented the above. If we're looking up by ID there's no need for further filtering. I think your comment confused me "all maintainers" but that's not what the query is doing.